packages/api/README.md

```diff
@@ -29,11 +29,12 @@ See [Cumulus README](https://github.com/cumulus-nasa/cumulus/blob/master/README.
 Running tests for kinesis-consumer depends on localstack. Once you have installed localstack, you can start it for dynamoDB only:
 
 ```
-SERVICES=dynamodb localstack start
+LAMBDA_EXECUTOR=docker localstack start
```

Comment:
Since we are running tests from the root level of the repo, we should add this to the main readme if necessary. Alternatively, we can add the env variable to the docker-compose.yml, to have it available as part of the `docker-compose up local` command.